modules/util/util.go

```diff
@@ -124,3 +126,28 @@ func MergeInto(dict map[string]interface{}, values ...interface{}) (map[string]i
 
 	return dict, nil
 }
+
+// RandomInt returns a random integer between 0 and limit, inclusive
+func RandomInt(limit int64) (int64, error) {
+	int, err := rand.Int(rand.Reader, big.NewInt(limit))
+	if err != nil {
+		return 0, err
+	}
+	return int.Int64(), nil
+}
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
```

Comment:
Why not use a const string like before?

Reply:
Hmm I just copied this from an example, need to check. Is there a performance benefit to doing it?

Reply:
done in https://github.com/go-gitea/gitea/pull/15741/commits/fbae1585eb03ddf28c155f2e53ec29947c6832b8.